config-connector/solutions/projects/kpt/owned-project/README.md

```diff
@@ -0,0 +1,66 @@
+Owned Project
+==================================================
+
+# NAME
+
+  owned-project
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create
+  a project in a folder, binding an IAM member
+  as project owner.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  Run `kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/projects/kpt/project-owner project-owner`.
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a
+  service account with the following roles in
+  the organization:
```

Comment:
As you said above, customers are creating the project in the folder, then the minimal required permissions would be at folder level instead of the organization level.